ec-europa/europa-component-library
# feat(preset): add corporate preset - INNO-917

# PR description

- `yarn dist` now generate css/js for all presets (not all preset use js)
- Add "corporate" preset containing only header and footer (and all dependencies)

How to test the corporate preset:
- Get or generate the css and js for corporate preset
- Use file `test-preset.html`attached to https://webgate.ec.europa.eu/CITnet/jira/browse/INNO-917
- Update path to europa-corporate.css and europa-corporate.js in the file
- display the html file; header and footer should be correctly displayed and funcitonal

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:
